pxr/pxrConfig.cmake.in

```diff
@@ -47,6 +47,16 @@ if(@PXR_ENABLE_PYTHON_SUPPORT@)
     endif()
 endif()
 
+include(CMakeFindDependencyMacro)
+
+if(@PXR_FIND_TBB_IN_CONFIG@)
+    find_dependency(TBB @TBB_VERSION@ CONFIG)
+endif()
+
+if(@PXR_FIND_OPENSUBDIV_IN_CONFIG@)
```

Comment:
Similar to above - can we replace with:

```
if (NOT DEFINED PXR_FIND_OPENSUBDIV_IN_CONFIG)
    set(PXR_FIND_OPENSUBDIV_IN_CONFIG "@PXR_FIND_OPENSUBDIV_IN_CONFIG@")
endif()
if(PXR_FIND_OPENSUBDIV_IN_CONFIG)
```

Reply:
9decb4e0fba8e83f34bbbe0814058ca72e912e1d